src/GitElephant/GitBinary.php

```diff
@@ -68,4 +73,18 @@ public function setPath(string $path)
     {
         $this->path = $path;
     }
+
+    /**
+     * @return string
+     */
+    public function getVersion()
+    {
+        if (is_null($this->version)) {
+
+            // unix only!
+            $this->version = exec('git --version | cut -d " " -f 3');
```

Comment:
Is this Unix-only because the `cut` command is - depending on the configuration - only available in Git Bash? Is so, could we make it cross-platform with e.g. `end(explode(" ". $output))`?